nixos/modules/services/monitoring/ups.nix

```diff
@@ -334,6 +335,7 @@ let
         MINSUPPLIES = lib.mkDefault 1;
         MONITOR = lib.flip lib.mapAttrsToList cfg.upsmon.monitor (name: monitor: with monitor; [ system powerValue user "\"@upsmon_password_${name}@\"" type ]);
         NOTIFYCMD = lib.mkDefault "${pkgs.nut}/bin/upssched";
+        POWERDOWNFLAG = "/run/killpower";
```

Comment:
It might work now, because of how /run/nut gets created. But I think it should be a RuntimeDirectory=, which means it'll be unmounted by the time the killpower service runs. We need to store the flag somewhere that outlives all normal services.

Note that upstream suggests /etc/killpower. I opted for /run because I prefer ramfs when that is possible. (And upsmon supports both volatile and non-volatile filesystems -- it'll remove the POWERDOWNFLAG on startup if it exists.)